src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,92 @@
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $white-color;
+
+  &__screen {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    width: 166px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 36px 0 4px;
+    text-align: left;
+  }
+
+  &__code {
+    font-size: 10px;
+    text-align: left;
+    color: $gray-color;
+    margin-top: 0;
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__btn {
+    width: 166px;
+    box-sizing: border-box;
+    display: inline-block;
+    border-radius: 5px;
+    border: none;
+    background: $blue-color;
+    color: $white-color;
+    padding: 12px 72px;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    text-decoration: none;
+    line-height: 16px;
+
+    &:hover {
+      border: 1px solid $blue-color;
+      background: $white-color;
+      color: $blue-color;
+      line-height: 14px;
+    }
+  }
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  width: 166px;
+  justify-content: space-between;
+  margin-bottom: 24px;
+
+  &-count {
+    font-size: 10px;
+    line-height: 14px;
+  }
+}
```

Comment:
use nesting

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/1c3c357e-f711-4914-8a5a-f23c6714af9c)
still not fixed, you have selector `card` already, use nesting
&__reviews